src/geo/projection/globe.js

```diff
@@ -41,6 +41,13 @@ export default {
     supportsFreeCamera: true,
     zAxisUnit: "pixels",
     center: [0, 0],
+    unsupportedLayers: new Set([
```

Comment:
Could we make this a simple array? And use `unsupportedLayers.includes(...)` instead?

Reply:
Sure!